src/test/device.test.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const server = require("../index");
+const request = require("supertest");
+
+const Device = require('../models/device.model');
+
+beforeAll(async () => {
+  const url = 'mongodb://localhost:27017/test-traceph';
```

Comment:
Will might run into problems if we integrate the test in our CI/CD workflow. Can we get the database url instead as an environment variable? Don't hesitate to add it or change the `.env.example`. 😄